java/arcs/sdk/android/storage/AndroidStorageServiceEndpointManager.kt

```diff
@@ -0,0 +1,20 @@
+package arcs.sdk.android.storage
+
+import android.content.Context
+import arcs.core.storage.DirectStorageEndpointManager
+import arcs.core.storage.StoreManager
+import arcs.sdk.android.storage.service.ConnectionFactory
+import kotlin.coroutines.CoroutineContext
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+
+@ExperimentalCoroutinesApi
+fun androidStorageServiceEndpointManager(
```

Comment:
I feel like this should be upper-cased, it's a pseudo-constructor and the existing pattern in both Kotlin stdlib, and our code-base, is to use upper-case.